### PR TITLE
subversion: add head

### PR DESCRIPTION
**Pull request by yurikoles**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/subversion.rb [hunk 1]
```diff
@@ -23,23 +30,23 @@ class Subversion < Formula
...
+  depends_on "apr-util"
 
   # Always build against Homebrew versions instead of system versions for consistency.
   depends_on "lz4"
+  depends_on "openssl"
   depends_on "sqlite"
   depends_on "utf8proc"
   depends_on "perl" => :recommended
 
   # For Serf
```
Comment (commitay) [inline Formula/subversion.rb]: This comment needs to be moved with the deps.

Comment (yurikoles) [reply Formula/subversion.rb]: those are fixes by audit, maybe better to remove comments?

**Comment by yurikoles** [comment]
@commitay thanks!

**Comment by commitay** [comment]
@yurikoles I've reverted the dependency order changes in this PR to avoid a conflict with https://github.com/Homebrew/homebrew-core/pull/32076

**Pull request merged**